src/org/labkey/test/tests/FileAttachmentColumnTest.java

```diff
@@ -15,153 +15,628 @@
         mouseOver(Locator.tagWithAttribute("img", "title", SAMPLE_JPG.getName()));
         shortWait().until(ExpectedConditions.presenceOfElementLocated(By.xpath("//div/span[contains(text(),'" + SAMPLE_JPG.getName() + "')]")));
         mouseOut();
     }
 
-    @Override
-    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    // sampleType
+    /*
+        This test exercises file fields but sets up the data by inserting each row individually
+     */
+    @Test
+    public void testSampleFileFields()
     {
-        _containerHelper.deleteProject(getProjectName(), afterTest);
+        SampleTypeDefinition subASampleType = new SampleTypeDefinition(SUBA_SAMPLETYPE_NAME)
+                .setFields(List.of(new FieldDefinition("color", ColumnType.String),
+                        new FieldDefinition("file", ColumnType.File)));
+        SampleTypeAPIHelper.createEmptySampleType(SUBFOLDER_A_PATH, subASampleType);
+
+        SampleTypeHelper.beginAtSampleTypesList(this, SUBFOLDER_A_PATH);
+        clickAndWait(Locator.linkWithText(SUBA_SAMPLETYPE_NAME));
+        DataRegionTable samplesRegion = new DataRegionTable("Material", getDriver());
+        List<String> fileNames = SAMPLE_FILES.stream().map(File::getName).toList();
+
+        for (File file : SAMPLE_FILES)
+        {
+            var queryUpdatePage = samplesRegion.clickInsertNewRow();
+            queryUpdatePage.setField("Name", file.getName());
+            queryUpdatePage.setField("file", file)
+                    .submit();
+        }
+        validateSampleData(SUBA_SAMPLETYPE_NAME, SUBFOLDER_A_PATH, SAMPLE_FILES);
+
+        var fileContentPage = FileContentPage.beginAt(this, SUBFOLDER_A_PATH);
+        fileContentPage.fileBrowserHelper().selectFileBrowserItem("/sampletype/csv_sample.csv");
+        var files = fileContentPage.fileBrowserHelper().getFileList(true);
+        checker().withScreenshot("unexpected_davfile_state")
+                .wrapAssertion(()-> Assertions.assertThat(files)
+                        .as("expect edited files to be present in dav locaiton")
+                        .containsExactlyInAnyOrderElementsOf(fileNames));
     }
 
-    @BeforeClass
-    public static void setupProject()
+    // assay
+    @Test
+    public void testAssayFileFieldsUI() throws Exception
     {
-        FileAttachmentColumnTest init = (FileAttachmentColumnTest)getCurrentTest();
-        init.doSetup();
+        String runName = "assay_run.tsv";
+        WebDavUploadHelper uploadHelper = new WebDavUploadHelper(SUBFOLDER_A_PATH);
+
+        // upload the files we'll be using as results so they can be resolved
+        for (File file : SAMPLE_FILES)
+        {
+            uploadHelper.uploadFile(file);
+        }
+        for (File file : OTHER_SAMPLE_FILES)
+        {
+            uploadHelper.uploadFile(file);
+        }
+
+        // generate some resultFiles
+        var resultFiles = new ArrayList<File>();
+
+
+        String runFileContent = """
```

Comment:
I meant that we could just use a file from the existing testAutomation/data folder so we don't have to generate those temp files